aidial_sdk/chat_completion/base.py

```diff
@@ -13,3 +19,13 @@ async def chat_completion(
 
     async def rate_response(self, request: RateRequest) -> None:
         """Implement rate response logic"""
+
+    async def tokenize(self, request: TokenizeRequest) -> TokenizeResponse:
+        """Implement tokenize logic"""
+        raise NotImplementedError()
```

Comment:
I want to make implementation of `tokenize` and `truncate_prompt` optional, so as to make this fix backward compatible.

If I don't raise the exception, then it leads to a type-checking error, because a method with an empty body is no-op method - it returns `None`. 

But these methods do not return `None` _(which makes them different from `rate_response` method btw)_. So I have to return something in a default implementation. Here I'm raising an exception.